### PR TITLE
feat: api routes for EvmApi

### PR DESCRIPTION
**Pull request by johnvsnagendra**
Suggestion to add separate api routes for each EvmAPI endpoint. 

The following files contain param types and nextjs API routes for the functions under `EvmApi.account`. 

By defining separate API routes for each EvmAPI endpoint, we can reuse them when calling from ClientSide or ServerSide use. 

Let me know your opinion on this suggestion.🙌

### CODE REVIEW

**Comment by Y0moo** [comment]
Hey @johnvsnagendra! We will release a library with API routes and hooks to call Moralis APIs from client-side

**Comment by Y0moo** [comment]
coming 2 weeks

File: src/types/EvmApi/account/types.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Account API Param Types
```
Comment (Y0moo) [inline src/types/EvmApi/account/types.ts:1]: Love this work with types. GJ on this

**Review by Y0moo — APPROVED** [review]

File: pages/api/EvmApi/account/getNFTTransfers.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+export default async function handler(
+  req: getNFTTransfersRequest,
+  res: NextApiResponse
+) {
+  await Moralis.start({ apiKey: process.env.MORALIS_API_KEY });
+
+  try {
+    const data = await Moralis.EvmApi.account.getNFTTransfers({
+      address: req.body.address,
```
Comment (Y0moo) [inline pages/api/EvmApi/account/getNFTTransfers.ts:17]: What'd say about destructurization like this here instead?
```
const { address, chain, providerUrl, toBlock } = req.body;
const data = await Moralis.EvmApi.account.getNativeBalance({ address, chain, providerUrl, toBlock });
```

**Review by Y0moo — CHANGES_REQUESTED** [review]

File: src/types/EvmApi/account/types.ts (hunk 1 above)
Comment (johnvsnagendra) [reply src/types/EvmApi/account/types.ts:1]: Glad you liked it. 
Do you think I should continue creating these API routes for other endpoints? If not, I can continue with the types, which I believe will be helpful even with the API routes library release.

File: pages/api/EvmApi/account/getNFTTransfers.ts (hunk 2 above)
Comment (johnvsnagendra) [reply pages/api/EvmApi/account/getNFTTransfers.ts:17]: Sure. I will make the required changes.

Comment (johnvsnagendra) [reply pages/api/EvmApi/account/getNFTTransfers.ts]: I have refreshed the files with the requested changes. You can check now.🙌

File: src/types/EvmApi/account/types.ts (hunk 1 above)
Comment (Y0moo) [reply src/types/EvmApi/account/types.ts:1]: @johnvsnagendra thanks. No need for manual writing them

**Pull request closed without merging**